packages/client/app/auth/forgot-password/confirmation-password/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
```

Comment:
let's move it to a separate folder instead of storing it inside the " forgot-password"

packages/client/app/auth/forgot-password/confirmation-password/page.tsx
--->
packages/client/app/auth/confirmation-password/page.tsx